manual/tests/Makefile

```diff
@@ -22,11 +26,30 @@ check-cross-references: cross-reference-checker
 	  $(TOPDIR)/driver/main_args.ml \
 	  $(TOPDIR)/lambda/translmod.ml
 
+# check that all standard library modules are referenced by the
+# standard library chapter of the manual
 .PHONY: check-stdlib
 check-stdlib:
 	./check-stdlib-modules $(TOPDIR)
 
 
+# check name collision between latex source file and module documentation
+# on case-insensitive file systems
+normalize = $(shell echo $(basename $(notdir $(1) )) | tr A-Z a-z)
+LOWER_MLIS= $(call normalize,$(DOC_ALL_MLIS))
+LOWER_ETEX= $(call normalize,$(wildcard $(MANUAL)/*/*.etex) $(wildcard *.etex))
+INTER = $(filter $(LOWER_ETEX), $(LOWER_MLIS))
+
+.PHONY: check-case-collision
+check-case-collision:
+ifeq ($(INTER),)
+else
+	@echo  "The following names\n  $(INTER)\nare used by both an OCaml module and \
+	a latex source file.\nThis creates a conflict on case-insensitive \
+	file systems." && false
```

Comment:
I have the smallest feeling that `\n` in an `echo` is not portable. Regardless, couldn't this just be a sequence of `@echo` commands culminating in an `@false`?

Reply:
You are right a sequence of `@echo` should be more portable, I will fix this point after the current CI run .